fonts/generate.py

```diff
@@ -118,7 +118,7 @@ def generate_smufl(opts: Namespace) -> bool:
 def extract_fonts(opts: Namespace) -> bool:
     """
     Takes a font file and extracts the necessary files for Verovio.
-    Generates a glyph file for each glyph,and an SVG glyph file
+    Generates a glyph file for each glyph, and an SVG glyph file
```

Comment:
This comma is not needed.

Reply:
Since there is no verb after the `and`, it is not an independent clause, so no comma is needed (or allowed by English teachers).